### PR TITLE
Avoid publishing graphs in migrations

### PR DESCRIPTION
**Pull request by jacobtylerwalls**
### Description of Change
Using a proxy model in a migration can create drift. Better to accomplish this with a `publish` command inside the `updateproject` command.

### Issues Solved
Follow-up to #9333

### Checklist

-   [ ] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: arches/management/commands/updateproject.py [hunk 1]
```diff
@@ -64,3 +65,8 @@ def update_to_v7(self):
                         graphs.append(graphid)
                 management.call_command("graph", operation="publish", update_instances=True, graphs=",".join(graphs))
                 cursor.execute("drop table if exists temp_graph_status")
+
+    def update_to_v7_5(self):
+        # Republish to pick up CardsXNodeXWidget changes from
+        # migration 10150_add_default_value_file_list.py
+        management.call_command("graph", operation="publish")
```
Comment (chiatt) [inline arches/management/commands/updateproject.py:72]: When I tested this it ended up publishing even unpublished graphs. There could be a reasons not to update the existing publications, but if we pass in the update flag we can prevent publishing unpublished models. 

I tried that just to confirm and found I got an integrity error if the graph was not already published, so I had to wrap it in a try block. 
```
from django.db.utils import IntegrityError

try:
    management.call_command("graph", operation="publish", update=True)
except IntegrityError:
    pass
```

Comment (jacobtylerwalls) [reply arches/management/commands/updateproject.py]: Ah got it, thanks. Added `--update` flag.

Re: the IntegrityError, I'd expect that to roll back the entire transaction. I'll see if I can do something about the root cause tomorrow?

**Comment by jacobtylerwalls** [comment]
I noticed the graph command only works on resource models, not branches. Is that okay here?

File: arches/management/commands/updateproject.py (hunk 1 above)
Comment (jacobtylerwalls) [reply arches/management/commands/updateproject.py]: #10410

**Review by chiatt — APPROVED** [review]
Looks good!

**Pull request merged**